Packs/GmailSingleUser/Integrations/GmailSingleUser/GmailSingleUser.py

```diff
@@ -49,11 +50,67 @@
     'Accept': 'application/json',
 }
 
-
+EXECUTION_METRICS = ExecutionMetrics()
 ''' HELPER FUNCTIONS '''
 
 
+def execute_gmail_action(service, action: str, action_kwargs: dict) -> dict:
```

Comment:
what's the type of `service`?

Reply:
we don't have a type hint for this...
Maybe we need to add stub lib for this
https://github.com/henribru/google-api-python-client-stubs 🤔

Reply:
True, but it's pretty much blocked by [CIAC-10534](https://jira-dc.paloaltonetworks.com/browse/CIAC-10534) so let's let it be for now.